src/main/kotlin/blackjack/domain/BlackjackGame.kt

```diff
@@ -1,28 +1,45 @@
 package blackjack.domain
 
-import blackjack.domain.player.Player
-import blackjack.domain.player.Players
+import blackjack.domain.gameresult.GameResults
+import blackjack.domain.participant.Dealer
+import blackjack.domain.participant.Player
+import blackjack.domain.participant.Players
 
 class BlackjackGame(
     private val deck: Deck,
     val players: Players,
+    val dealer: Dealer,
 ) {
     fun start() {
         players.drawInitCards(deck)
+        dealer.drawCards(deck.pullOutFirstTurn())
     }
 
-    fun isPlaying(): Boolean {
+    fun isPlayerTurn(): Boolean {
         return players.isExistWaitingPlayer()
     }
 
     fun askDrawToCurrentTurnPlayer(isDrawCard: Boolean) {
-        val player = players.findCurrentTurnPlayer()
+        val player = findCurrentTurnPlayer()
         if (isDrawCard) {
             player.drawCard(deck.pullOut())
             return
         }
         player.endOwnTurn()
     }
 
-    fun findCurrentTurnPlayer(): Player = players.findCurrentTurnPlayer()
+    fun findCurrentTurnPlayer(): Player = players.findCurrentTurnPlayer() as Player
+
+    fun isSatisfiedDealerPullOutCondition(): Boolean {
+        val drawable = dealer.isAbleToDraw()
+        if (drawable) {
+            dealer.drawCard(deck.pullOut())
+        }
+
+        return drawable
```

Comment:
drawable이라는 변수 없이 제공하는것도 괜찮은 방법이라고 생각이 드네요!

```suggestion
        if (dealer.isAbleToDraw()) {
            dealer.drawCard(deck.pullOut())
            return true
        }
        return false
```

Reply:
`drawable` 변수 추출하는게 엄청 애매모호하다고 생각했는데, (결과적으로는 drawable을 전달하는 것과 같지만) 결과의 성공 여부를 반환하는 것이 더 자연스러운 것 같네요. 👍